libsolidity/codegen/ir/IRGenerationContext.cpp

```diff
@@ -32,6 +32,16 @@ using namespace solidity;
 using namespace solidity::util;
 using namespace solidity::frontend;
 
+FunctionDefinition const* IRFunctionGenerationQueue::pop()
+{
+	if (m_definitions.empty())
```

Comment:
Would it be better to rather assert `!m_definitions.empty()`?

Reply:
Yeah, good idea. I added it before I added `empty()` and originally I wanted to check the result. But now I don't need to.